backend/src/packages/chats/chat.controller.ts

```diff
@@ -163,13 +178,39 @@ class ChatController extends BaseController {
    *                   type: array
    *                   items:
    *                     $ref: '#/components/schemas/Chat'
+   *       400:
+   *         description: Bad Request
+   *         content:
+   *           application/json:
+   *             schema:
+   *               type: object
+   *               properties:
+   *                 status:
+   *                   type: integer
+   *                   example: 400
+   *                 message:
+   *                   type: string
+   *                   example: The data which was passed is incorrect.
    */
   private async findAll(
-    options: APIHandlerOptions<{ user: UserAuthResponseDto }>,
+    options: APIHandlerOptions<{
+      user: UserAuthResponseDto;
+      query: GetChatsByQueryDto;
+    }>,
   ): Promise<APIHandlerResponse> {
+    if (!checkQuery(options.query.query) && options.query.query) {
```

Comment:
schema: {
      querystring: validationSchema,
},
Just in this way

Reply:
done